modules/rosa-hcp-creating-account-wide-sts-roles-and-policies.adoc

```diff
@@ -26,15 +26,11 @@ Before using the {product-title} (ROSA) CLI (`rosa`) to create {hcp-title-first}
 ----
 $ rosa create account-roles --hosted-cp
 ----
-+
-[source,terminal]
-----
-$ ACCOUNT_ROLES_PREFIX="${ACCOUNT_ROLES_PREFIX}"
-----
-+
+
+* Optional: Set your account role prefix as an environmental variable with the following command: 
 [source,terminal]
 ----
-$ rosa create account-roles --hosted-cp --prefix $ACCOUNT_ROLES_PREFIX
```

Comment:
Does this command get run somewhere else? I'm just seeing the removal here, but not the addition anywhere else. Wondering if it was just a previously overlooked redundancy that you're fixing?

Reply:
Ya, so it doesn't actually work this way. When I executed the command, it still asked for me to enter a prefix. I figured, this could cause issues for our users. Though, maybe it's missing an `=` between `--prefix` and the variable?

Reply:
Unfortunately, I think that's "just the behavior" of our CLI, i.e., it still confirms flags in interactive mode (for better or worse ¯\_(ツ)_/¯). With that said, you _should_ be able to bypass with

`rosa create account-roles --hosted-cp --prefix $ACCOUNT_ROLES_PREFIX --yes` or you might need `rosa create account-roles --hosted-cp --prefix $ACCOUNT_ROLES_PREFIX --mode auto --yes`

Reply:
```
--yes                           Automatically answer yes to confirm operation.
```

Reply:
`--mode auto --yes` should do the trick but that command is mandatory to create the account roles